alot/commands/envelope.py

```diff
@@ -375,12 +375,13 @@ def openEnvelopeFromTmpfile():
         old_tmpfile = None
         if self.envelope.tmpfile:
             old_tmpfile = self.envelope.tmpfile
-        self.envelope.tmpfile = tempfile.NamedTemporaryFile(delete=False,
-                                                            prefix='alot.',
-                                                            suffix='.eml')
-        self.envelope.tmpfile.write(content.encode('utf-8'))
-        self.envelope.tmpfile.flush()
-        self.envelope.tmpfile.close()
+        with tempfile.NamedTemporaryFile(delete=False, prefix='alot.',
+                                         suffix='.eml') as tmpfile:
+            tmpfile.write(content.encode('utf-8'))
+            # Is this needed if the file is closed at the end of the with
+            # block?
+            tmpfile.flush()
```

Comment:
I would leave it, flush is basically a blocking operation to ensure that the file moves out of the harddrive cache and is actually written to disk (which is basically a  no-op on SSDs and NVMe).